designdocs/wallettool-subcommands.adoc

```diff
@@ -0,0 +1,514 @@
+= Wallettool Documentation
```

Comment:
I suggest "WalletTool Conversion to PicoCLI Subcommands"

The design document should describe the proposed change.